assets/helpers/flashSale.js

```diff
@@ -85,7 +85,7 @@ function getIntcmp(product: SubscriptionProduct, intcmp: ?string, defaultIntcmp:
   return intcmp || defaultIntcmp;
 }
 
-function getEndTime(): number { return new Date(2018, 9, 24, 14, 30, 0).getTime(); }
+function getEndTime(): number { return new Date(2018, 10, 4, 0, 0, 0).getTime(); }
```

Comment:
Can you use the same date as is used in the `flashSaleIsActive()` function above? Then we'll only need to change the date in one place next time we do this.

Reply:
Great point!